server/types/bytea_array.go

```diff
@@ -14,9 +14,7 @@
 
 package types
 
-import (
-	"github.com/lib/pq/oid"
-)
-
 // ByteaArray is the array variant of Bytea.
-var ByteaArray = createArrayType(Bytea, SerializationID_ByteaArray, oid.T__bytea)
+var ByteaArray = CreateArrayTypeFromBaseType(Bytea)
+
+// createArrayType(Bytea, SerializationID_ByteaArray, oid.T__bytea)
```

Comment:
We can remove this too (and the same commented out code in the other array files too).